ansible/roles/operator-pipeline/tasks/webhook-event-listener.yml

```diff
@@ -0,0 +1,107 @@
+---
+- name: Create event listener allowing triggering the pipelines via Github webhook
+  tags:
+    - webhook
+  block:
+    - name: Create event listener
+      k8s:
+        state: present
+        namespace: "{{ oc_namespace }}"
+        definition:
+          apiVersion: triggers.tekton.dev/v1alpha1
+          kind: EventListener
+          metadata:
+            name: operator-pipeline-github-listener
+            labels:
+              app: operator-pipeline
+              suffix: "{{ suffix }}"
+              env: "{{ env }}"
+          spec:
+            serviceAccountName: pipeline
+            triggers:
+              # run Hosted pipeline on PR opened, reopened, synchronized
+              - name: github-pull-request-listener
+                interceptors:
+                  - github:
+                      secretRef:
+                        secretName: github-webhook-secret
+                        secretKey: webhook-secret
+                      eventTypes:
+                        - pull_request
+                  - cel:
+                      filter: >-
+                        (
+                          header.match("X-GitHub-Event", "pull_request")
+                          && body.action in ["opened", "reopened", "synchronize"]
+                          && body.pull_request.base.ref == "{{ branch }}"
+                        )
+                bindings:
+                  - ref: operator-hosted-pipeline-trigger-binding
+                template:
+                  ref: operator-hosted-pipeline-trigger-template
+              # run Hosted pipeline on PR labeled with label "pipeline/trigger"
```

Comment:
Do we need a similar listener for the release pipeline? I'm assuming this is for manual intervention in the case of a failure so it may be useful to have the same for the release pipeline because we won't be able to change the PR state.

Reply:
Done, there are now listeners for labels pipeline/trigger-hosted and pipeline/trigger-release